### PR TITLE
Mrsteve galaxy update, linting fixes, warning controls added, patch/audit added to control names, And New readme Added, Added workflows

### PR DESCRIPTION
**Pull request by MrSteve81**
Adding Galaxy Update And Proper Issue Templates

### CODE REVIEW

File: .github/workflows/new_contrib_message.yml
Comment (uk-bolly) [inline .github/workflows/new_contrib_message.yml:1]: The action should be main not 1.1.0

File: .github/workflows/testing_pipeline.yml
Comment (uk-bolly) [inline .github/workflows/testing_pipeline.yml:1]: Discord URL needs updating

File: .github/workflows/windows.tfvars
Comment (uk-bolly) [inline .github/workflows/windows.tfvars:1]: Maybe change prefix to match Linux pipelines?

File: .github/workflows/new_contrib_message.yml
Comment (MrSteve81) [reply .github/workflows/new_contrib_message.yml:1]: Added

File: .github/workflows/testing_pipeline.yml
Comment (MrSteve81) [reply .github/workflows/testing_pipeline.yml:1]: Updated in 2 places

File: .github/workflows/windows.tfvars
Comment (MrSteve81) [reply .github/workflows/windows.tfvars:1]: Changed the testing_pipeline.yml to have runs-on: windows

**Review by georgenalen — APPROVED** [review]
Looks good!

**Pull request merged**